### DISCUSSION TITLE
Bad compared to others

### DISCUSSION THREAD

**Discussion by Havouza15** [matiasdelellis/facerecognition#723] [General]
I have setup this app and scanned about 500 images. 11 faces found when I counted on command line output. But no faces show up. I also notised that the cpu usage was very high. I have a 6 core AMD. 
When I compare with Immich that I also have tested, the Immich wins a knockout. All faces recognized and shown and grouped. Why is it so big difference

**Comment by ewok2** [comment]
Maybe because there are some parameter that alow a fine tunning... and this fine tunning leed if not well adjust to a bad result ;-)

**Comment by matiasdelellis** [comment]
Hi @Havouza15 
> I have setup this app and scanned about 500 images. 11 faces found when I counted on command line output. But no faces show up.

This seems to be very little, but we would have to see the logs and the configuration you are using to give an opinion. 
Maybe image formats that nextcloud doesn't support, or maybe small images that are ignored, or probably an imaginary problem.

> When I compare with Immich that I also have tested, the Immich wins a knockout. All faces recognized and shown and grouped. 

I didn't have the opportunity to try Immich but it is a great project. From what I see they use the insightface model for face detection, which is not especially better than dlib's.

> Why is it so big difference

Don't know. Maybe it is a good model to try to make another external model. All the pieces are there. 😉

**Reply by hobbycharken** [reply]
well I have an Immich server up for those who want an cheap account. It works great and seems very stable.
